CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to Linuxbrew
 
-First time contributing to Linuxbrew? Read our [Code of Conduct](https://github.com/Linuxbrew/brew/blob/master/CODE_OF_CONDUCT.md#code-of-conduct).
+First time contributing to Linuxbrew? Read our [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODE_OF_CONDUCT.md#code-of-conduct) and review [How To Open a Homebrew Pull Request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request).
```

Comment:
Was the `Linuxbrew` to `Homebrew` change  intentional here?

Reply:
Yes. I believe the two code of conducts are identical, so I prefer to link to upstream's document.